extensions/swift-package-index/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "swift-package-index",
+  "title": "Swift Package Index",
+  "description": "Search for Swift Packages via https://swiftpackageindex.com",
+  "icon": "command-icon.png",
+  "author": "maximkrouk",
+  "owner": "capturecontext",
```

Comment:
@maximkrouk please remove the owner property, we don't add it for public extensions